build.xml

```diff
@@ -583,8 +583,7 @@
           <dependency groupId="com.googlecode.json-simple" artifactId="json-simple" version="1.1"/>
           <dependency groupId="com.boundary" artifactId="high-scale-lib" version="1.0.6"/>
           <dependency groupId="com.github.jbellis" artifactId="jamm" version="${jamm.version}"/>
-
-          <dependency groupId="org.yaml" artifactId="snakeyaml" version="1.23"/>
+          <dependency groupId="org.yaml" artifactId="snakeyaml" version="1.26"/>
```

Comment:
sadly this isn't enough as we commit the binary into the repo: see https://github.com/apache/cassandra/tree/trunk/lib

Reply:
Added in another commit to add the library.

Reply:
Just for my understanding, why we are maintaining libraries in the codebase?

Reply:
I don't know, it was this way before I joined.  For whatever reason test libs are pulled from maven central but src libs are not and pulled from local disk.